### PR TITLE
Fixes bodybag examine text

### PR DESCRIPTION
**Pull request by alsoandanswer**
It was incorrectly using closet examine code and it incorrectly kept displaying it as empty.

This should resolve that issue.

Also tides up code here and there and some cleanup.

PR'd again, in an attempt to stop travis cloning itself.

Also makes a proc out of the examine to make things cleaner I guess

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!bugfix

File: code/game/objects/items/bodybag.dm [hunk 1]
```diff
@@ -37,11 +38,24 @@
...
+		to_chat(user, "\The [src] is empty.")
+	else if(storage_capacity > content_size*4)
+		to_chat(user, "\The [src] is barely filled.")
+	else if(storage_capacity > content_size*2)
+		to_chat(user, "\The [src] is less than half full.")
+	else if(storage_capacity > content_size)
+		to_chat(user, "\The [src] still has some free space.")
+	else
+		to_chat(user, "\The [src] is full.")
+	to_chat(user, "It [contains_body ? "contains" : "does not contain"] a body.")
```
Comment (skull132) [inline code/game/objects/items/bodybag.dm:57]: uuh. Didn't you want this this to just say whether it contains a body or not, and not mess around with the filled-o-meter? If so, literally remove all of the lines but 57.

Comment (alsoandanswer) [reply code/game/objects/items/bodybag.dm:57]: Nah, I still want the filled-o-meter.

Cause you know, bodybags still can hold things _other_ than the body.

**Pull request merged**